backend/server.py

```diff
@@ -1,15 +1,34 @@
-from stt_model import get_stt_from_path
-from fastapi import FastAPI, UploadFile, Form
+from fastapi import FastAPI, UploadFile, Form, Depends
+from fastapi.middleware.cors import CORSMiddleware
 from fastapi.responses import JSONResponse
+from fastapi.security import OAuth2PasswordRequestForm
 from dtos import Document
 from pathlib import Path
 import shutil
 import data.mongo as repo
 import os
 import pathlib
+import os
+from login import register_user, login, oauth2_scheme, google_login
+from config_loader import *
+import os
+from pydantic import BaseModel
+from typing import Optional, List
+from user_types import UserRegister
 
```

Comment:
I don't like this if is here, maybe it might be better to create a separate imports file and import it, or think of another solution

Reply:
its only for types validation. no logic on the controller class at all. I couldn't manage to find how to do multiple contorller files in fastapi but i can if you'd prefer